src/xdocs/config_coding.xml

```diff
@@ -2515,6 +2515,8 @@ public native void myTest(); // violation
                 STRING_LITERAL</a>
                 , <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#CHAR_LITERAL">
                 CHAR_LITERAL</a>
+                , <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#TEXT_BLOCK_CONTENT">
+                TEXT_BLOCK_CONTENT</a>
```

Comment:
please move this update in separate issue, as we will need test is separately.

Reply:
Fixed.

Reply:
Please share link to issue where you plan to do this.

Reply:
https://github.com/checkstyle/checkstyle/issues/8576